.github/workflows/cla.yml

```diff
@@ -0,0 +1,80 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened, closed, synchronize]
+
+jobs:
+  CLAssistant:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "CLA Assistant"
+        if: (github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'
+        uses: cla-assistant/github-action@v2.1.3-beta
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PERSONAL_ACCESS_TOKEN: ${{ secrets.CLA_ACTION_ACCESS_TOKEN }}
+        with:
+          # Where to store the signatures
+          remote-organization-name: gitpod-io
+          remote-repository-name: cla
+          branch: "main"
+          path-to-signatures: "signatures/v1/cla.json"
+          # messages for the bot to post
+          custom-notsigned-prcomment: |
+            Thank you for your submission, we really appreciate it.
+
+            Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://gitpod-io.github.io/cla/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
+
+            If you want to contribute on behalf of a company rather than as an individual please leave a comment stating so and we will get in touch.
+          custom-pr-sign-comment: "I have read the CLA Document and I hereby sign the CLA"
+          # The following GitHub users don't have to sign the CLA
+          # - Gitpod employees (https://github.com/orgs/gitpod-io/teams/team-gitpod/members)
+          # - Bots
+          # - Anyone who signed the CLA manually prior to the use of CLA Assistant
+          #   Filip Troníček (@filiptronicek)
+          #   Jordan Hailey (@jordanhailey)
+          #   Ofer Shaal (@shaal),
+          #   Pudong Zheng (@iQQBot)
+          allowlist: >
```

Comment:
thoughts: drop all people from this list except robots. Use this list only for robots which cannot do the /sign process. Asking folks to resign isn't a concern is it?

Reply:
@ghuntley I'll remove all the existing signers (it is probably out of date already anyway). For the bots  I'll keep them for the first iteration so we can get this out the door and then we can simplify it later.